k9mail/src/main/res/layout/openpgp_inline_dialog.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:padding="10dp">
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_marginBottom="10dp"
+        tools:ignore="UseCompoundDrawables"
+        >
+
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="20dp"
+            android:layout_gravity="center_vertical"
+            android:src="@drawable/compatibility"
+            android:scaleType="fitCenter"
+            android:tint="@color/light_black"
+            />
```

Comment:
It would be nice if we could move towards setting `android:contentDescription` (http://developer.android.com/training/accessibility/accessible-app.html) or `tools:ignore="contentDescription"` for ImageView elements.

Personally I'd like us to move towards builds failing on lint warnings. But in the meantime it would be nice if we didn't make more of them :)

I don't mind it going in without this given there's so many already though. We can swing through the whole lot adding them.